src/javascripts/lib/i18n.js

```diff
@@ -0,0 +1,68 @@
+import manifest from '../../manifest.json'
+
+// map to store the key/translation pairs of the loaded language
+let translations
+
+/**
+ * Replace placeholders in the given string with context
+ * @param {String} str string with placeholders to be replaced
+ * @param {Object} context object contains placeholder/value pairs
+ * @return {String} formatted string
+ */
+function parsePlaceholders (str, context) {
+  const regex = /{{(.*?)}}/g
+  const matches = []
+  let match
+
+  do {
+    match = regex.exec(str)
+    if (match) matches.push(match)
+  } while (match)
+
+  return matches.reduce((str, match) => {
+    const newRegex = new RegExp(match[0], 'g')
+    str = str.replace(newRegex, context[match[1]] || '')
+    return str
+  }, str)
+}
+
+class I18n {
+  constructor (locale) {
```

Comment:
Yes!, default `en` local is set on the `loadTranslations` function below.

Reply:
I feel the constructor should have that same default because just because we only use it in 1 spot for now, doesn't mean this code will stay this way.